docs/hooks.md

```diff
@@ -1,356 +1,165 @@
-# How to use hooks
+## Overview
 
-TODO:
+Hooks are a common way to extend systems. Let's say you want to send a 
+message to someone in your organization every time a contact is created. An 
+easy way to do this would be to insert code to send the message in the 
+CiviCRM core code right where the contact is created. However, as soon as we 
+upgrade to a newer version all this code will be overwritten. This is where 
+hooks come in to save the day.
 
--   http://wiki.civicrm.org/confluence/display/CRMDOC/Hooks
--   http://wiki.civicrm.org/confluence/display/CRMDOC/Hook+Reference
+At key points in processing - for example saving 
+something to the database - CiviCRM checks to see whether you've "hooked in" 
+some custom code, and runs any valid code it finds.
 
-Hooks are a common way to extend systems. The way they work in CiviCRM is that,
-at key points in processing - such as saving a change or displaying a page -
```

Comment:
OK, I don't know if i'm going to be able to read everything, but just want to give some validation: Yes-yes for jumping into the example first and removing silly words (like "the way they work in CiviCRM is s that"). :+1: